client/src/i18n/en-US.json

```
@@ -372,7 +376,8 @@
     "submissions": {
       "submitter": {
         "title": "Your Submissions",
-        "byline": "Submissions you have submitted for review"
+        "byline": "Submissions you have submitted for review",
+        "tooltip": "{type_name} with a status of Draft, Initially Submitted, or Awaiting Review will not appear here."
```

```suggestion
        "tooltip": "The submissions of others will not appear here."
```
Submissions with those statuses _will_ appear within submissions tables for submitters.